Jenkinsfile

```diff
@@ -3,24 +3,82 @@ pipeline {
         label 'docker'
     }
     stages {
-        stage ('Docker') {
-            agent {
-                dockerfile {
-                    args '-e HOME=/tmp -e BUILD_CONTEXT=ci -e CCACHE_DIR=/ccache -v $HOME/.ccache:/ccache'
-                    reuseNode true
+        stage('Parallel') {
+            parallel{
+                stage('GCC'){
+                    agent {
+                        dockerfile {
+                            additionalBuildArgs '--target native-gcc'
+                            args '-e HOME=/tmp -e BUILD_CONTEXT=native-gcc -e CCACHE_DIR=/ccache -v $HOME/.ccache:/ccache'
+                            reuseNode true
+                        }
+                    }
+                    stages {
+                        stage("Build") {
+                            steps {
+                                sh '''
+                                SCRUTINY_BUILD_TEST=1 \
+                                SCRUTINY_BUILD_TESTAPP=1 \
+                                scripts/build.sh
+                                '''
+                            }
+                        }
+                        stage("Test") {
+                            steps {
+                                sh '''
+                                scripts/runtests.sh
+                                '''
+                            }
+                        }
+                    }
                 }
-            }
-            stages {
-                stage ('Build') {
-                    steps {
-                        sh 'scripts/build.sh'
+                stage('Clang'){
+                    agent {
+                        dockerfile {
+                            additionalBuildArgs '--target native-clang'
+                            args '-e HOME=/tmp -e BUILD_CONTEXT=native-clang -e CCACHE_DIR=/ccache -v $HOME/.ccache:/ccache'
+                            reuseNode true
+                        }
+                    }
+                    stages {
+                        stage("Build") {
+                            steps {
+                                sh '''
+                                CMAKE_TOOLCHAIN_FILE=cmake/clang.cmake \
+                                SCRUTINY_BUILD_TEST=1 \
+                                SCRUTINY_BUILD_TESTAPP=1 \
+                                SCRUTINY_WERR=0 \
```

Comment:
CI builds should always have WERR enabled

Reply:
Werr enabled by default now